embrace-android-features/src/main/kotlin/io/embrace/android/embracesdk/internal/anr/sigquit/SigquitDataSourceImpl.kt

```diff
@@ -46,13 +46,9 @@ internal class SigquitDataSourceImpl(
             val res = sigquitNdkDelegate.installGoogleAnrHandler(googleThreadId)
             if (res > 0) {
                 googleAnrTrackerInstalled.set(false)
-                logger.logError("Could not initialize Google ANR tracking {code=$res}")
```

Comment:
I thought about logging an internal error here, but we weren't tracking these before, so I decided against it. Happy to add it if y'all think it's worthwhile